contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -72,10 +72,12 @@ namespace eosiosystem {
 
       if( usecs_since_last_fill > 0 && _gstate.last_pervote_bucket_fill > time_point() ) {
          auto new_tokens = static_cast<int64_t>( (continuous_rate * double(token_supply.amount) * double(usecs_since_last_fill)) / double(useconds_per_year) );
+         const uint64_t days_in_gbm_period = useconds_in_gbm_period / useconds_per_day;
+         const uint64_t days_remaining_in_gbm_period = (gbm_final_time - current_time_point()).count();
          // needs to be 1/2 Savings, 1/6 Producers, 1/6 Voters, 1/6 Genesis Block Member
          auto to_voters        = new_tokens / 6;
          auto to_per_block_pay = to_voters;
-         auto to_gbm           = to_voters;
+         auto to_gbm           = to_voters * 2 * (double(days_remaining_in_gbm_period) / double(days_in_gbm_period));
          auto to_savings       = new_tokens - (to_voters + to_per_block_pay + to_gbm);
```

Comment:
Since the continuous rate does not account for the gbm tokens being added, we will have to add the to_gbm amount to new_tokens above this line

Reply:
Let's see if this is what you mean, Aaron.

Reply:
yessir this is it :)